src/newmagic.cpp

```diff
@@ -1441,11 +1447,18 @@ void cast_combat_spell(struct char_data *ch, int spell, int force, char *arg)
     SET_WAIT_STATE_AND_COMBAT_STATUS_AFTER_OFFENSIVE_SPELLCAST;
 
     success = success_test(skill, GET_BOD(vict) + target_modifiers);
+    snprintf(rbuf, sizeof(rbuf), "successes: %d", success);
+    act(rbuf, TRUE, ch, NULL, NULL, TO_ROLLS);
     if (success > 0 && GET_REFLECT(vict) && (reflected = reflect_spell(ch, vict, spell, force, 0, GET_BOD(ch), success))) {
       vict = ch;
       ch = temp;
     }
     success -= resist_spell(vict, spell, force, 0);
+    snprintf(rbuf, sizeof(rbuf), "reflected: %s", reflected ? "yes" : "no");
+    act(rbuf, TRUE, ch, NULL, NULL, TO_ROLLS);
+    success -= resist_spell(vict, spell, force, 0);
```

Comment:
You've doubled up the resist_spell check here, so it's getting resisted twice-- did you mean to delete the previous one at line 1456?

Reply:
Yuuuuup, good catch